lib/stdlib/src/encode_zstd.rs

```diff
@@ -88,34 +88,3 @@ impl FunctionExpression for EncodeZstdFn {
         TypeDef::bytes().infallible()
     }
 }
-
-#[cfg(test)]
-mod test {
-    use super::*;
-    use base64::Engine;
-
-    fn decode_base64(text: &str) -> Vec<u8> {
-        let engine = base64::engine::GeneralPurpose::new(
-            &base64::alphabet::STANDARD,
-            base64::engine::general_purpose::GeneralPurposeConfig::new(),
-        );
-
-        engine.decode(text).expect("Cannot decode from Base64")
-    }
-
-    test_function![
-        encode_zstd => EncodeZstd;
-
-        with_defaults {
-            args: func_args![value: value!("you_have_successfully_decoded_me.congratulations.you_are_breathtaking.")],
-            want: Ok(value!(decode_base64("KLUv/QBY/QEAYsQOFKClbQBedqXsb96EWDax/f/F/z+gNU4ZTInaUeAj82KqPFjUzKqhcfDqAIsLvAsnY1bI/N2mHzDixRQA").as_bytes())),
```

Comment:
Could this maybe be changed to assert things like:
* output size is different (smaller) than input
* output has zstd header
* output can be decoded back to the input with `decode_zstd`